arm-redis/2016-04-01/swagger/redis.json

```diff
@@ -794,6 +913,98 @@
         "files"
       ],
       "description": "Parameters for redis import operation."
+    },
+    "ScheduleEntry": {
+      "properties": {
+        "dayOfWeek": {
+          "type": "string",
+          "description": "Day of week when cache can be patched.",
+          "enum": [
+            "Monday",
+            "Tuesday",
+            "Wednesday",
+            "Thursday",
+            "Friday",
+            "Saturday",
+            "Sunday"
+          ],
+          "x-ms-enum": {
+            "name": "DayOfWeek",
+            "modelAsString": true
```

Comment:
Would you ever want to modify this enum? I don't think so. If not then, `modelAsString: false`. This will generate an Enum in languages like C#, Java that support enums and in javascript it will validate against the set of allowed values.